File: app/src/main/java/com/neome/api/meta/base/dto/StudioDtoRowIdPointer.java
// neome.ai API - do not change
//

package com.neome.api.meta.base.dto;

import com.neome.api.meta.base.Types.MetaIdSpreadsheet;

import org.jetbrains.annotations.Nullable;

@SuppressWarnings("unused")
public class StudioDtoRowIdPointer extends StudioBase
{
  @Nullable
  public StudioBuildArgBinder rowId;

  @Nullable
  public MetaIdSpreadsheet spreadsheetId;
}

File: app/src/main/java/com/neome/api/meta/base/dto/StudioDtoRowIdPointer.kt
// neome.ai API - do not change
//
@file:Suppress("unused", "MemberVisibilityCanBePrivate", "CanBeParameter", "PropertyName")


package com.neome.api.meta.base.dto

import com.neome.api.meta.base.Types.MetaIdSpreadsheet

class StudioDtoRowIdPointer : StudioBase() {
    var rowId: StudioBuildArgBinder? = null
    var spreadsheetId: MetaIdSpreadsheet? = null
}
